tests/namespace_tests.py

```diff
@@ -322,9 +334,13 @@ def __test(value):
 
     # note: 1j should also be invalid, but currently not caught
     # by the schema validation and hence removed from the test
-    for value in ['a', None, True]:
+    # for value in ['a', None, True]:
+
+    # None and True cast to float
+    for value in ['a']:
         yield __test, value
```

Comment:
in pytest, but jams is not yet switched over to pytest.  We'll handle that in #122